features/step_definitions/common_steps.rb

```diff
@@ -266,7 +266,11 @@
   begin
     banner_message = page.find(:css, ".dm-banner, .banner-#{status}-without-action", wait: false)
   rescue Capybara::ElementNotFound => e
-    banner_message = page.find(:css, ".banner-#{status}-with-action", wait: false)
+    begin
+      banner_message = page.find(:css, ".banner-#{status}-with-action", wait: false)
+    rescue Capybara::ElementNotFound => e
+      banner_message = page.find(:css, ".dm-alert--#{status}", wait: false)
+    end
```

Comment:
Might be easier to do

```suggestion
    banner_message = page.find(:css, ".dm-alert--#{status}, .banner-#{status}-with-action", wait: false)
```